src/realtime_metrics/main.py

```diff
@@ -347,8 +349,8 @@ def availability_acceptable_stop_time_updates(stop_time_updates: list[tuple[Trip
 
     Parameters:
     stop_time_updates: list of corresponding trip updates and stop time updates
-    time_frame_start: start of the time frame to observe, in minutes since 1970
-    time_frame_end: end of the time frame to observe, in minutes since 1970
+    time_frame_start: start of the time frame to observe, in seconds since 1970
+    time_frame_end: end of the time frame to observe, in seconds since 1970
 
     Returns:
     A float containing the percentage of one minute slots with two or more updates.
```

Comment:
Docstring seems not to be correclty formatted: 
![grafik](https://github.com/user-attachments/assets/afe6e157-1f23-4819-a64d-c90aa67e8e8a)

Reply:
I adapted the format.